File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        // -100 <= nums[i] <= 100
        int val = -101;
        int j = 0;
        for(int i = 0 ; i < nums.size(); ++i){
            if(nums[i] != val){
                nums[j++] = nums[i];
                val = nums[i];
            }
        }
        return j;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let val = -101;
    let j = 0;
    for(let i = 0 ; i < nums.length; ++i){
        if(nums[i] !== val){
            nums[j++] = nums[i];
            val = nums[i];
        }
    }
    return j;
};